README.md

```diff
@@ -34,21 +32,18 @@ In order to use this plugin you need "perf" to be installed on a Linux target ho
 
 ### System Requirements
 
-* root privileges
-* "perf" installed on a host
-* /proc/sys/kernel/perf_event_paranoid set to 0 (in other case list of metrics can be limited)
-* Linux kernel version at least 2.6.31
-* [golang 1.5+](https://golang.org/dl/) (needed only for building)
```

Comment:
Seems like we don't include this info anymore so if I decided to make more complex README.md update I tried to unify as much as possible.